### PR TITLE
Rock blue input elements

### PR DESCRIPTION
**Pull request by ramshorst**

### CODE REVIEW

**Comment by ramshorst** [comment]
Ready for review! @AlexisMontagne @tgallice @phndiaye

File: less/base/_images.less [hunk 1]
```diff
@@ -16,3 +16,9 @@
   height: 96px;
   border-radius: 48px;
 }
+
+.upf-image--round-288 {
+  width: 288px;
+  height: 288px;
+  border-radius: 144px;
+}
```
Comment (phndiaye) [inline less/base/_images.less]: 👍 👍 👍 👍

File: less/base/_inputs.less [hunk 2]
```diff
@@ -78,17 +78,33 @@
...
+  box-shadow: none;
 
   &:focus {
     .remove-outline;
-    border: 1px solid @upf-primary-dark-purple;
+    border: 1px solid @upf-gray-dark;
+  }
+
+  &::placeholder {
+  color: @upf-primary-rock-blue-light;
```
Comment (phndiaye) [inline less/base/_inputs.less]: no ident here :/

File: less/base/_inputs.less [hunk 3]
```diff
@@ -78,17 +78,33 @@
 // This class can be applied on Input elements, Textareas, or selects.
 //
 .upf-input {
-  border: 1px solid @upf-gray-dark;
+  border: 1px solid @upf-gray;
```
Comment (phndiaye) [inline less/base/_inputs.less]: does it change much ? `upf-gray-dark` is used pretty much everywhere as the "standard" border color... does it change much to change it ?

File: less/base/_transitions.less [hunk 4]
```diff
@@ -0,0 +1,16 @@
+/*
+ * Transitions
+ * All transitions.
+ */
+
```
Comment (phndiaye) [inline less/base/_transitions.less]: you can remove that line :)

**Comment by ramshorst** [comment]
I think it does but of course this is arguable!

**Pull request merged**